it/mail/chrome/messenger/webservices/pbWebServices.properties

```diff
@@ -1,15 +1,15 @@
 # Flickr currently, explore reuse with other services in bug 3281
-webservice.errorTitle=%1$S Error
-webservice.authFailedTryAgain=Login failed. Please try again, ensuring you completed all steps.
-webservice.notPermitted=Your %1$S account doesn't have permission to use this feature.
-webservice.noFrob=Login Failed. Please try again.
-webservice.unexpected=Unexpected response from server. Please ensure that Postbox is authorized to use %1$S.
-webservice.internalError=A server error occurred on %1$S.
-webservice.requestFail=A request to %1$S has failed. Please try again.
+webservice.errorTitle=Errore %1$S
+webservice.authFailedTryAgain=Accesso fallito. Riprova, assicurandoti di aver completato tutti gli step.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+webservice.authFailedTryAgain=Accesso fallito. Riprova, assicurandoti di aver completato tutti i passaggi.